docs/docs/transactions.md

```diff
@@ -84,11 +91,13 @@ sequelize.transaction(function (t1) {
 });
 ```
 
+### Nested/Partial transactions
```

Comment:
No nesting here - two separate transactions. This example is menat to show how you can override the CLS transaction parameter by passing another transaction.

It might actually be benefical to highlight that the transactions are _not_ nested... Maybe concurrent transactions ?